shuup/reports/writer.py

```diff
@@ -122,6 +124,51 @@ def get_filename(self, report):
         return "%s%s" % ((report.filename_template % fmt_data).replace(":", "_"), self.extension)
 
 
+class CSVReportWriter(ReportWriter):
+    content_type = "text/csv"
+    extension = ".csv"
+    writer_type = "csv"
+
+    def __init__(self):
+        super(CSVReportWriter, self).__init__()
+        self.data = []
+
+    def _w(self, content):
+        if content is not None:
+            if isinstance(content, TaxlessPrice) or isinstance(content, TaxfulPrice):
+                content = floatformat(content.amount.value, 2)
```

Comment:
you can't do this. Each currency has it's own decimal precision. you should:
```py
float(content.amount.as_rounded().value)
```
also fix this in the `ExcelReportWriter`